### PR TITLE
Added items 4951-4969

### PR DESCRIPTION
**Pull request by Lemongrass3110**
* **Addressed Issue(s)**: None

* **Server Mode**: Renewal

* **Description of Pull Request**: 
Added a few missing enchants.

A few known issues:
1. 4954,Earth_Drop: google translates the name to: wood drop
1. 4955,Holy_Drop: google translates the name to: optical/light drop
1. 4956,Recovery_Drop: find the items/item group
1. 4969,Libra: find the items/item group

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -3119,6 +3119,25 @@
...
+4956,Recovery_Drop,Recovery Drop,6,0,,,,,,,,,,,,,,,{ setarray .@skills$, "AL_HEAL", "PR_SANCTUARY", "AM_POTIONPITCHER", "AB_HIGHNESSHEAL", "AB_CHEAL"; for( .@i = 0; .@i < getarraysize(.@skills$); .@i++ ){ bonus2 bSkillHeal,.@skills$[.@i],2; bonus2 bSkillHeal2,.@skills$[.@i],2; } /* TODO: Depending on some recovery items HP recovery amount + 2% */ },{},{}
+4957,Famitsus_Power,Famitsu's Power,6,0,,,,,,,,,,,,,,,{ bonus bMaxHP,832; },{},{}
+4958,Gemini,Gemini,6,0,,,,,,,,,,,,,,,{ bonus bDelayrate,1; },{},{}
+4959,Sagittarius,Sagittarius,6,0,,,,,,,,,,,,,,,{ bonus bLongAtkRate,1; },{},{}
+4960,Aquarius,Aquarius,6,0,,,,,,,,,,,,,,,{ bonus bUseSPrate,-2; },{},{}
+4961,Aries,Aries,6,0,,,,,,,,,,,,,,,{ bonus bMatk,10; },{},{}
+4962,Cancer,Cancer,6,0,,,,,,,,,,,,,,,{ bonus bAtk,3; },{},{}
+4963,Taurus,Taurus,6,0,,,,,,,,,,,,,,,{ bonus bAtk,4; bonus bHit,1; },{},{}
+4964,Capricorn,Capricorn,6,0,,,,,,,,,,,,,,,{ bonus2 bMagicAddRace,RC_All,3; },{},{}
+4965,Pisces,Pisces,6,0,,,,,,,,,,,,,,,{ bonus bCastrate,-2; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: In renewal
`bonus bCastrate,n;             		Skill cast time rate + n%. (If RENEWAL_CAST is defined, this bonus is equal to bVariableCastrate)`

File: db/re/item_db.txt [hunk 2]
```diff
@@ -3119,6 +3119,25 @@
...
+4952,Fire_Drop,Fire Drop,6,0,,,,,,,,,,,,,,,{ bonus2 bAddEle,Ele_Fire,2; bonus2 bMagicAtkEle,Ele_Fire,2; bonus2 bSubEle,Ele_Fire,1; },{},{}
+4953,Water_Drop,Water Drop,6,0,,,,,,,,,,,,,,,{ bonus2 bAddEle,Ele_Water,2; bonus2 bMagicAtkEle,Ele_Water,2; bonus2 bSubEle,Ele_Water,1; },{},{}
+4954,Earth_Drop,Earth Drop,6,0,,,,,,,,,,,,,,,{ bonus2 bAddEle,Ele_Earth,2; bonus2 bMagicAtkEle,Ele_Earth,2; bonus2 bSubEle,Ele_Earth,1; },{},{}
+4955,Holy_Drop,Holy Drop,6,0,,,,,,,,,,,,,,,{ bonus2 bAddEle,Ele_Holy,2; bonus2 bMagicAtkEle,Ele_Holy,2; bonus2 bSubEle,Ele_Holy,1; },{},{}
+4956,Recovery_Drop,Recovery Drop,6,0,,,,,,,,,,,,,,,{ setarray .@skills$, "AL_HEAL", "PR_SANCTUARY", "AM_POTIONPITCHER", "AB_HIGHNESSHEAL", "AB_CHEAL"; for( .@i = 0; .@i < getarraysize(.@skills$); .@i++ ){ bonus2 bSkillHeal,.@skills$[.@i],2; bonus2 bSkillHeal2,.@skills$[.@i],2; } /* TODO: Depending on some recovery items HP recovery amount + 2% */ },{},{}
+4957,Famitsus_Power,Famitsu's Power,6,0,,,,,,,,,,,,,,,{ bonus bMaxHP,832; },{},{}
+4958,Gemini,Gemini,6,0,,,,,,,,,,,,,,,{ bonus bDelayrate,1; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: `bonus bDelayrate,-1;`

File: db/re/item_combo_db.txt [hunk 3]
```diff
@@ -404,9 +404,21 @@
 4681:4691,{ autobonus "{ active_transform 3241,6000; bonus bAtk,100; }",30,6000,BF_WEAPON; }
 4682:4694,{ autobonus "{ active_transform 3244,6000; bonus bAtk,100; }",30,6000,BF_WEAPON; }
 4683:4695,{ autobonus "{ active_transform 3246,6000; bonus bAtk,100; }",30,6000,BF_WEAPON; }
-5040:5442,{ bonus bAspdRate,3; bonus bVariableCastrate,3; }
 4697:27012,{ bonus bNoMadoFuel,1; }
 4699:27012,{ bonus bNoMadoFuel,1; }
+4958:20212,{ .@r = getequiprefinerycnt(EQI_COSTUME_HEAD_TOP); if( .@r >= 7 ){ bonus bDelayrate,-1 - ( .@r > 9 ) ? 1 : 0; } }
```
Comment (Atemo) [inline db/re/item_combo_db.txt]: Top costume can be refined now?

Comment (Lemongrass3110) [reply db/re/item_combo_db.txt]: Uhm I do not really know if this is possible now, but that was the only item I found that would match the description. Maybe there is a non costume version of that item?

File: db/re/item_db.txt [hunk 4]
```diff
@@ -3119,6 +3119,25 @@
 4948,Skill_Delay1,After Skill Delay Lv1,6,10,,,,,,,,,,,,,,,{ bonus bDelayrate,-2; },{},{}
 4949,Skill_Delay2,After Skill Delay Lv2,6,10,,,,,,,,,,,,,,,{ bonus bDelayrate,-4; },{},{}
 4950,Skill_Delay3,After Skill Delay Lv3,6,10,,,,,,,,,,,,,,,{ bonus bDelayrate,-6; },{},{}
+4951,Darkness_Drop,Darkness Drop,6,0,,,,,,,,,,,,,,,{ bonus2 bAddEle,Ele_Dark,2; bonus2 bMagicAtkEle,Ele_Dark,2; bonus2 bSubEle,Ele_Dark,1; },{},{}
```
Comment (Atemo) [inline db/re/item_db.txt]: I'm sceptical about the magical effect.
- Increases physical damage on dark monster.
- Reduces damage from dark attack.
- **Increases damage of dark element magic**. There is no magic skill element dark (for player) if I'm not wrong. Also the previous effects apply vs dark monsters but this one deal more damage vs monsters of opposite element in the table of element.

The scripts match the description. Wrong description / translation ?

Comment (Akaineko-astasi) [reply db/re/item_db.txt]: There is actually 1 dark magic skill: Hell Inferno.

Comment (Lemongrass3110) [reply db/re/item_db.txt]: I will merge it like that for now.
We can change it later on anyway. :-)

**Pull request merged**